example/heatEquation2D/src/heatEquation2D.cpp

```diff
@@ -215,5 +215,7 @@ auto main() -> int
     //   TagCpuSerial, TagGpuHipRt, TagGpuCudaRt, TagCpuOmp2Blocks, TagCpuTbbBlocks,
     //   TagCpuOmp2Threads, TagCpuSycl, TagCpuTbbBlocks, TagCpuThreads,
     //   TagFpgaSyclIntel, TagGenericSycl, TagGpuSyclIntel
+
+    alpaka::printAccTags<alpaka::EnabledAccTags>();
```

Comment:
Please add this to all examples, this behaviour is not only affecting the heat equation